README-cn.md

```diff
@@ -6,12 +6,13 @@
 ## 翻译:
 
 * [English](README.md)
-* [Spanish](README-es.md) 
+* [Español](README-es.md)
+* [Français](README-fr.md)
 * [中文版](README-cn.md)
 * [日本語](README-ja.md)
 
```

Comment:
Not sure about that. People are more like to recognize their language if written in their mother tongue. It's also the way it's done in wikipedia.